Dockerfile

```diff
@@ -69,9 +70,6 @@ ENV HOME="/root" \
   VIRTUAL_ENV=/lsiopy \
   PATH="/lsiopy/bin:$PATH"
 
-# copy sources
-COPY sources.list /etc/apt/
```

Comment:
What was this doing/why don't we need it anymore?

Reply:
so the original dockerfile was building a base image from quite literally absolute scratch. it had nooooothing on it. our ubuntu base image already has a list of apt sources that are fips approved/compliant to use. so we dont need to add more. actually we shouldnt add more.